.jenkins/jenkins.sh

```diff
@@ -22,21 +22,25 @@
 # JOB_URL            Full URL of this job, like http://server:port/jenkins/job/foo/
 
 # stop on all errors
-set +e
+set -e
```

Comment:
this was just a typo before?

Reply:
I was not sure myself, but the fact that the regression tests run through indicates to me that the author of this actually wanted to switch ON error checking.